resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -62,32 +68,15 @@ x64
 
 ## GitHub
 
-In MDS we will use [GitHub.com](https://github.com/) as well as an Enterprise version of GitHub hosted here at UBC, [GitHub.ubc.ca](https://github.ubc.ca). Please follow the set-up instructions for both below.
+In MDS we will use the publicly available [GitHub.com](https://github.com/) as well as an Enterprise version of GitHub hosted here at UBC, [GitHub.ubc.ca](https://github.ubc.ca). Please follow the set-up instructions for both below.
 
 #### GitHub.com
-If you do not yet have one, sign up for a free account at https://github.com/ .
 
-#### GitHub.ubc.ca
-For us to add you to the MDS organization on [Github.ubc.ca](https://github.ubc.ca) we need you to login using your CWL:
-
-visit [Github.ubc.ca](https://github.ubc.ca) to do this.
-
-This step is required for
-- being able to store your work
-- all homework submission and grading
-- working collaboratively
-
-## GitHub
-
-In MDS we will use [GitHub.com](https://github.com/) as well as an Enterprise version of GitHub hosted here at UBC, [GitHub.ubc.ca](https://github.ubc.ca). Please follow the set-up instructions for both below.
-
-#### GitHub.com
-If you do not yet have one, sign up for a free account at [GitHub.com](https://github.com/).
+Sign up for a free account at https://github.com/ if you don't have one already.
```

Comment:
Not sure that https://github.com/ will show up as a hyperlink on the website, you might need to change to [https://github.com/](https://github.com/) or something like that.

Reply:
I have a slight preference for visible link + hyperlink to make it super obvious its a link they should click on. I sometimes miss a hidden coloured link that you only see because it is coloured. This might be especially problematic for colourblind or visually impaired folks as well...

Reply:
@ttimbers Agree that only color is not sufficient. I updated to either include the link or mentioned "read more" or similar together with a descriptive name.

For the future, I think a good accessibility practice is using a color of a different contrast/luminosity and underlining links. Regarding text, I believe that short and descriptive is preferred over the actual URL due to increased accessibility for people using screen readers, which read urls letter by letter, and functionalities such as compiling a list of links are not great to listen to (same goes when using only "read more" etc), and navigating by voice is also really hard with long URLs.

Unless you know of someone in this year's cohort that has indicated that they are using a screen reader, maybe we can implement this next year? I opened issue https://github.com/UBC-MDS/UBC-MDS.github.io/issues/59 for this.

Reply:
Sounds good. And no, I am not aware yet of a student who is colour blind or has other visual impairments. I just remember in my ugrad course, a colour blind student was so grateful that we taught the use of colour blind friendly palettes that this has stuck with me. Let's go with this for now but keep that issue open as a reminder to do better next year.